versioned/persist/nontx/src/main/java/org/projectnessie/versioned/persist/nontx/NonTransactionalDatabaseAdapterConfig.java

```diff
@@ -20,6 +20,7 @@
 
 public interface NonTransactionalDatabaseAdapterConfig extends DatabaseAdapterConfig {
   int DEFAULT_PARENTS_PER_GLOBAL_COMMIT = 50;
+  int DEFAULT_GLOBAL_LOG_ENTRY_SIZE = 250_000;
```

Comment:
What do you think about a two-stage lookup where the global log contains small entries with keys of global objects? The first read would load a map of content ID -> blob ID and the second look up would load the actual data for a particular content ID. This way we won't have to read all global contents when we need to access only one content ID and at the same time we still cap the number of read for each content ID.

Reply:
I suspect that's a refactoring of the data model for the global log.
Not sure whether that's worth it, given that the global log isn't needed anymore when (Iceberg) metadata is in Nessie.